web-ng/src/app/services/layer/layer.service.ts

```diff
@@ -125,10 +125,25 @@ export class LayerService {
    */
   private static createFormMap(id: string, form: Form): Map<string, Form> {
     let forms = Map<string, Form>();
+    if (LayerService.isFormFieldEmpty(form)) {
```

Comment:
Logic that deals with forms doesn't belong in the service; this should be in the UI/form component.

Reply:
Also, we don't want to create a form if the field is empty, but looking at this code it here it appears we do create the form {} in the db.